generator/cli/generate-migration-scripts.ts

```diff
@@ -76,7 +76,7 @@ export default async function generateMigrationScripts(params: {
       if (!content) throw new Error(`Could not read schema at path ${SchemaPathTemplateOriginal}`)
 
       const schema = Remeda.pipe(
-        content,
+        mysqlSchemaTypeTransformUtil(content, combination.datasourceProvider),
```

Comment:
This can be made into a curried function or wrapped in an anonymous function in the pipe. Either way it will solve the problem of the pipe symmetry being broken here. It should start with a value and then be a chain of functions.